### PR TITLE
update security suppliedBy.md range to point to core suppliedby.md

### PR DESCRIPTION
**Pull request by jeff-schutt**
3rd change to resolve https://github.com/spdx/spdx-3-model/issues/436 

Intent is to have the suppliedBy property in the security profile reuse the suppliedBy property defined in the Core profile to avoid duplication of definitions.

### CODE REVIEW

**Comment by jeff-schutt** [comment]
In #436 I see Gary made a different suggestion [here](https://github.com/spdx/spdx-3-model/issues/436#issuecomment-1673619564). Review requested to determine what change is needed.

File: model/Security/Classes/VulnAssessmentRelationship.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ assessment relationships. It factors out the common properties shared by them.
   - minCount: 0
   - maxCount: 1
 - suppliedBy
```
Comment (goneall) [inline model/Security/Classes/VulnAssessmentRelationship.md:30]:
```suggestion
- /Core/suppliedBy
```

File: model/Security/Classes/VulnAssessmentRelationship.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ assessment relationships. It factors out the common properties shared by them.
...
-  - type: /Core/Agent
+  - type: /Core/SuppliedBy
```
Comment (goneall) [inline model/Security/Classes/VulnAssessmentRelationship.md:31]:
```suggestion
  - type: /Core/Agent
```

**Review by goneall — CHANGES_REQUESTED** [review]

**Review by goneall — APPROVED** [review]
LGTM

**Comment by tsteenbe** [comment]
Discussed in the Defects meeting of Sept 6th, 2023 by @goneall  @jeff-schutt  @rnjudge @tsteenbe - MarkDown encoding for this pull request is now correct but spec parser needs to be updated by @goneall  before this can be merged.

**Comment by goneall** [comment]
Turns out I was wrong on the analysis of the CI failure - this is failing due to the Jinja changes - not due to handling /Core.

I tested this change locally on my machine with an earlier version of the spec-parser and it works, so this is safe to merge.

**Comment by goneall** [comment]
Just a note: this is a fix for the CI failure: https://github.com/spdx/spec-parser/pull/69

**Review by zvr — APPROVED** [review]

**Comment by rnjudge** [comment]
LGTM

**Review by kestewart — APPROVED** [review]
Reviewed on call.   Looks good.

**Comment by goneall** [comment]
To be merged once we merge the new parser - cc: @zvr

**Comment by goneall** [comment]
The parser has been updated - merging...

**Pull request merged**